### DISCUSSION TITLE
Is it possible to create a popup window?

### DISCUSSION THREAD

**Discussion by raiguard** [kovidgoyal/kitty#4018] [Q&A] [answered]
I am finally biting the bullet and working to improve my [kakoune](https://github.com/mawww/kakoune) workflow, and since kak does not do any windowing, I'm doing all of my windowing with kitty. I would very much like to avoid using `tmux` or something similar, because that adds yet another layer of complexity to the issue that I do not wish to mess with, and it breaks some of kak's visual features (specifically, colored and curly underlines).

Occasionally, I would like to have a popup window. This could be for an ongoing process, a fuzzy finder, or a git manager. I want to stay "in context" so I don't want it to take up the whole screen with the `stack` layout.

Is this possible? I haven't found anything about popup windows on the site, and couldn't find any issues either. I assume the layout would have to support them somehow.

Here's an example of a popup in`tmux`:

![image](https://user-images.githubusercontent.com/3515394/132972203-39a828cd-5fe0-4927-a170-a178fd0a8b21.png)

**Comment by kovidgoyal** [comment, marked as answer]
You can create overlay windows these are not quite floating windows, since they overlay the entire underlying window, and are thus similar to stack layout windows, however they dont require you to change the layout and therefore mess with your context. So they should be fine for this use case.

True floating windows are not currently possible.

**Reply by raiguard** [reply]
Thanks. It's a shame that true popup windows aren't possible, but I will look into overlay windows. Consider this my official feature request for a `floating` window type.
